modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -19193,6 +19193,10 @@ this-configuration-is-inherited=This configuration is inherited.
 this-configuration-is-not-saved-yet=This configuration is not saved yet.
 this-configuration-is-not-saved-yet.-the-values-shown-are-the-default=This configuration is not saved yet. The values shown are the default.
 this-configuration-is-read-only=This configuration is read only.
+this-contact-does-not-have-any-additional-email-addresses=This contact does not have any additional email addresses.
```

Comment:
Can we remove the -additional- part? Because the account has only these, compared with the user that has its "default" one and these in concact

Reply:
One question: what is using the user page for texts? Are not the same?

Reply:
the user pages are using text "this-user-does-not-have-any-*". I was not sure if it made since for it to be "this-user*". Let me know what you think, if its ok, I can remove the new ones and use the same ones as the user pages.

Reply:
oh I noticed organizations has "this-organiztion-does-not-*, maybe itd make more sense to have the account one to be "this-account-does-not-*.